docs/doc/15-sql-functions/111-system-functions/fuse_encoding.md

```diff
@@ -0,0 +1,55 @@
+---
+title: FUSE_ENCODING
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.162"/>
+
+Returns the encoding types applied to a specific column within a table. It helps you understand how data is compressed and stored in a native format within the table.
```

Comment:
Does this table func only support `native` storage foramt? @SkyFan2002

Reply:
Yes, I will add parquet support after migrate arrow2 to arrow-rs and support adaptive encoding for parquet